tests/utils.hpp

```diff
@@ -17,6 +18,26 @@ namespace oxen::quic
 
     inline auto test_cat = oxen::log::Cat("test");
 
+    namespace test::defaults
+    {
+        inline const std::string CLIENT_SEED{
+                oxenc::from_hex("468e7ed2cd914ca44568e7189245c7b8e5488404fc88a4019c73b51d9dbc48a5"sv)};
+        inline const std::string CLIENT_PUBKEY{
+                oxenc::from_hex("626136fe40c8860ee5bdc57fd9f15a03ef6777bb9237c18fc4d7ef2aacfe4f88"sv)};
+        inline const std::string SERVER_SEED{
+                oxenc::from_hex("fefbb50cdd4cde3be0ae75042c44ff42b026def4fd6be4fb1dc6e81ea0480c9b"sv)};
+        inline const std::string SERVER_PUBKEY{
+                oxenc::from_hex("d580d5c68937095ea997f6a88f07a86cdd26dfa0d7d268e80ea9bbb5f3ca0304"sv)};
```

Comment:
I'm thinking that it might be nice to have this randomize the keypairs on each initialization; with libsodium that's trivial, but I'm not sure if gnutls gives us something nice to do that or not.

Reply:
So we could use `nettle` for this (we already have or build it, as it's a dependency of gnutls), but I'm not sure whether it's worth worrying about right now.  The fixed pregenerated keys are probably okay.

Reply:
Good idea. I messed with the `gnutls_rnd` function a bit but it was fruitless in this regard. Gnutls also has specific methods for generating ed keys, but it has to be directly into their abstract types like `gnutls_privkey_t`